.github/workflows/build-test-vm.yaml

```diff
@@ -65,15 +67,21 @@ jobs:
           cache: false
         # Sometimes setup-go gets stuck. Without this, it'll keep going until the job gets killed
         timeout-minutes: 10
-      - run: make docker-build-daemon
+      - name: Build daemon image
+        run: make docker-build-daemon
+        env:
+          IMG_DAEMON: ${{ steps.tags.outputs.daemon }}
+
       - run: make bin/vm-builder
+        env:
+          IMG_DAEMON: ${{ steps.tags.outputs.daemon }}
 
       - name: upload vm-builder
         if: ${{ format('{0}', inputs.upload-vm-builder) == 'true' }}
         uses: actions/upload-artifact@v4
         with:
           name: vm-builder
-          path: bin/vm-builder
+          path: ${{ format('bin/vm-builder-{0}', matrix.arch) }}
```

Comment:
question: Does this mean we need to change `release.yaml`'s handling of vm-builder?

Reply:
https://github.com/neondatabase/autoscaling/issues/1250

Reply:
fixed